src/lib/setup.js

```diff
@@ -85,7 +85,7 @@ export default function (app) {
   }
 
   // Timeout middleware
-  if (isProduction) {
+  if (process.env.NODE_ENV === "production") {
```

Comment:
Is extracting this out in such a verbose way necessary? We've never had any problems with this in the past.

Reply:
It was required to remove the `environment.ts` file. `isProduction` and `isDevelopment` were not used in enough places to warrant their own file. Additionally, I'm planning to separate the development entry point from the production entry point so that we don't deploy webpack as part of the server bundle. Which would remove all of these checks.